### DISCUSSION TITLE
We need an editable Notes tab in the App Store apps!

### DISCUSSION THREAD

**Discussion by C-Fu** [runtipi/runtipi-appstore#5752] [Ideas]
So in the new(er) Proxmox, for each VM/LXC you can add notes, which is not only very much welcomed, but very very important to quickly keep track of things related to the LXC/VM like...... username and passwords of (docker) apps that you installed.
Especially if you just created the LXC/VM or even some containers just to try out.
Example:

![image](https://github.com/user-attachments/assets/cc84b32e-bb77-4345-8132-81c101caaaef)

So what if there's an extra tab called Notes?

![image](https://github.com/user-attachments/assets/8d97d988-f772-437e-b1ef-1272f6fdc610)

This way, if the container asks you to put in a new username and password you can immediately write it down to the notes section right there, next to Logs, Backups etc?

Other tab suggestions:
- Shell
- docker-compose.yml export of the App that we deployed (under My Apps of course)
- Container performance using docker stats container_name

Take a look at some apps like glances && lazydocker for the dashboard, maybe you can find things that can be interesting to include. Keep up the awesome work dudes and dudettes!

**Comment by JigSawFr** [comment]
@nicotsx @steveiliop56 fyi
